docs/en/data_types/nested_data_structures/simpleaggregatefunction.md

```diff
@@ -0,0 +1,39 @@
+# SimpleAggregateFunction(name, types\_of\_arguments…) {#data-type-simpleaggregatefunction}
+
+Unlike `AggregateFunction`:
+
+- `SimpleAggregateFunction` is stateless, so it supports only simple stateless functions and for now, only the following are supported:
+
+    - `any`
+    - `anyLast`
+    - `min`
+    - `max`
+    - `sum`
+    - `sumMap`
+    - `groupBitAnd`
+    - `groupBitOr`
+    - `groupBitXor`
+
+- Type of the `SimpleAggregateFunction(func, Type)` is `Type` itself, so you do not need to apply functions with `-Merge`/`-State` suffixes.
+- Hence has better performance.
+
+**Parameters**
+
+-   Name of the aggregate function.
+-   Types of the aggregate function arguments.
```

Comment:
needs an example with multiple arguments

Reply:
None of functions supports multiple arguments (sumMap had been dropped, since it is not supported yet)